File: examples/todo-java-spring/output/src/main/java/com/example/todo/FavouriteTodoRepository.java
package com.example.todo;

import org.springframework.stereotype.Repository
import org.springframework.data.jpa.repository.JpaRepository
@Repository
public interface FavouriteTodoRepository extends JpaRepository<FavouriteTodo, java.util.UUID> {
    java.util.List<FavouriteTodo> findByFavouriteTrue();
    java.util.List<FavouriteTodo> findByFavourite(boolean favourite);
}

File: examples/todo-kotlin-quarkus/output/src/main/kotlin/com/example/todo/FavouriteTodoRepository.kt
package com.example.todo

import jakarta.enterprise.context.ApplicationScoped
import io.quarkus.hibernate.orm.panache.PanacheRepository
@ApplicationScoped
open class FavouriteTodoRepository : PanacheRepository<FavouriteTodo> {
    fun findByFavourite(favourite: Boolean): java.util.List<FavouriteTodo> {
        return find("favourite", favourite).list()
    }
}